src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -46,22 +49,36 @@
                 xhr.send(payload || '');
             },
 
-            hasClass = function(el, klass) {
-                return el.className && el.className.match(new RegExp('\\b' + klass + '\\b'));
+            hasClass = function(el, cssClass) {
+                if (el.classList) {
```

Comment:
I can't use the given code (*Uncaught SyntaxError: Unexpected token if*). Please note that these methods are in fact properties of `Sfjs` object.